src/app/exercise/exercise/exercise.base.ts

```diff
@@ -72,7 +72,8 @@ export class ExerciseBase implements OnDestroy {
   constructor(public slide: SlideComponent, private monacoConfig: MonacoConfigService) {
     this.onActiveUsubscribe = slide.onActive.filter(a => a).subscribe(() => {
       console.log('ACTIVE');
-      slide.disableShortcuts();
+
+      // slide.disableShortcuts();
```

Comment:
Ok sounds good to me. We should remove this line if it's not being used rather than leaving a comment. That is what led me to ask! Thanks!